jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/core/listeners/HtmlReporter.java

```diff
@@ -115,4 +117,27 @@ public void testEnded(String host) {
 
   }
 
+  /**
+   * Allows to configure APDEX for all requests in test
+   *
+   * @param satisfied - "satisfied" threshold
+   * @param tolerated - "tolerated threshold
+   * @return HtmlReporter
+   *
+   * @since 0.57
+   */
+  public HtmlReporter adpex(Duration satisfied, Duration tolerated) {
+
+    JMeterUtils.setProperty(
```

Comment:
As is, this might fail (with a NullPointerException) or not get expected results. Consider setting the properties within buildTestElement or buildTreeUnder which is assured to execute within a JMeter context. You can see an example in DslViewResultsTree.resultsLimit